packages/desktop-client/src/components/modals/ImportTransactions.js

```diff
@@ -339,18 +339,15 @@ function SubLabel({ title }) {
 
 function SelectField({ width, style, options, value, onChange }) {
   return (
-    <Select
+    <CustomSelect
+      options={[
+        ['', 'Choose field...'],
+        options.map(option => [option, option]),
```

Comment:
I think this needs a spread so that all of the options are shown as separate options.

```suggestion
        ...options.map(option => [option, option]),
```